test/check-application

```diff
@@ -1938,7 +1940,7 @@ class TestApplication(testlib.MachineCase):
         # Pruning again, should delete all system images
         b.click("#image-actions-dropdown")
         b.click("button:contains(Prune unused images)")
-        b.wait_js_func("ph_count_check", ".pf-c-modal-box__body .pf-c-list li", 3)
+        b.wait_js_func("ph_count_check", ".pf-c-modal-box__body .pf-c-list li",self.system_images_count - 1 if self.machine.ostree_image else self.system_images_count)
```

Comment:
missing space after comma, isn't flake8 going to complain?

Reply:
heh, it does, very loudly. I applaud your efforts to regularly validate our CI for that! :+1:

Reply:
Ofcourse :(